telco-examples/edge-clusters/README.md

```diff
@@ -0,0 +1,172 @@
+
+- `${BMC_USERNAME}` - The username for the BMC of the new Baremetal host.
+- `${BMC_PASSWORD}` - The password for the BMC of the new Baremetal host.
+- `${BMC_MAC}` with the MAC address of the new Baremetal host to be used.
+- `${BMC_ADDRESS}` with the URL for the baremetal Host BMC. If you want to know more about the different options available depending on your hardware provider, please check the following [link](https://github.com/metal3-io/baremetal-operator/blob/main/docs/api.md).
+
+Then, you need to apply the changes using the following command into the management cluster:
+
+```
+$ kubectl apply -f bmh-example.yaml
+```
+
+The new Baremetal host will be enrolled changing the state from registering, inspecting and available. You could check the status using the following command:
+
+``` 
+$ kubectl get bmh
+```
+
+#### Provision the new host using the CAPI manifests and the image generated
+
+Once the new Baremetal host is available, you need to provision the new host using the CAPI manifests and the image generated in the previous step.
+
+The first thing is to modify the `telco-capi-single-node.yaml` file and replace the following with your values:
+
+- `${EDGE_CONTROL_PLANE_IP}` - The IP address to be used as a endpoint for the edge cluster (should match with the kubeapi-server endpoint).
+- `${SRIOV_VENDOR}` - The vendor of the SRIOV network device to be used (e.g `8086` which means intel. You can get that info using `lspci` command with grep to find the vendors and device codes). For more information, please check the following [link](https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin#configurations)
+- `${SRIOV_DEVICE}` - The device of the SRIOV network device to be used (e.g `57c1` which means the FEC acc card. You can get that info using `lspci` command with grep to find the vendors and device codes). For more information, please check the following [link](https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin#configurations)
+- `${SRIOV_NET_INTERFACE}` - The network interface to be used for the SRIOV network device (e.g `eth0` which means the first network interface in the server. You can get that info using `ip link` command to list the network interfaces).
+- `${ISOLATED_CPU_CORES}` - The isolated CPU cores to be used for workloads pinning some specific cpu cores. You could get that info using `lscpu` command to list the CPU cores and then, select the cores to be used for the edge cluster in case you need cpu pinning for your workloads. For example, `1-18,21-38` could be used for the isolated cores.
+- `${NON-ISOLATED_CPU_CORES}` - The cores listed could be used shared for the rest of the process running on the edge cluster. For example, `0,20,21,39` could be used for the non-isolated cores.
+- `${CPU_FREQUENCY}` - The frequency to be used for the CPU cores. For example, `2500000` represent 2.5Ghz configuration and it could be used to set the CPU cores to the max performance.
```

Comment:
```suggestion
- `${CPU_FREQUENCY}` - The frequency to be used for the CPU cores. For example, `2500000` represents 2.5Ghz configuration and it could be used to set the CPU cores to the max performance.
```

Reply:
Thanks !!!